src/main/java/sg/ncl/MainController.java

```diff
@@ -1525,6 +1560,70 @@ public String editTeamProfile(
         return "redirect:/team_profile/" + teamId;
     }
 
+    @RequestMapping(value = "/team_quota/{teamId}", method = RequestMethod.POST)
+    public String editTeamQuota(
+            @PathVariable String teamId,
+            @ModelAttribute("teamQuota") TeamQuota editTeamQuota,
+            final RedirectAttributes redirectAttributes,
+            HttpSession session) throws IOException {
+
+        JSONObject teamQuotaJSONObject = new JSONObject();
+        teamQuotaJSONObject.put("teamId", teamId);
+
+        //check if budget input is positive
+        if (Double.parseDouble(editTeamQuota.getBudget()) < 0) {
+            redirectAttributes.addFlashAttribute("editBudget", "negativeError");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "editBudget" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)